apps/server/src/modules/common-cartridge/common-cartridge-client/courses-client.module.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { TestingModule, Test } from '@nestjs/testing';
+import { CoursesClientModule } from './courses-client.module';
+import { CoursesClientAdapter } from './courses-client.adapter';
+
+describe('CommonCartridgeClientModule', () => {
+	let module: TestingModule;
+
+	beforeAll(async () => {
+		module = await Test.createTestingModule({
+			imports: [
+				CoursesClientModule.register({
+					basePath: 'http://localhost:3000',
+				}),
+			],
+		}).compile();
+	});
+
+	afterAll(async () => {
+		await module.close();
+	});
+
+	it('should be defined', () => {
```

Comment:
The 'when' sentence is missing.

```suggestion
	describe('when the module is initialized', () => {
```

Reply:
done ✅